src/pybind/mgr/dashboard_v2/.pylintrc

```diff
@@ -135,8 +135,7 @@ disable=print-statement,
         bad-continuation,
         trailing-comma-tuple,
         unused-import,
-        fixme,
-        logging-format-interpolation
+        fixme
```

Comment:
Can we please drop the 'fixme' as well? Even if this makes the linter results noisier, if we have `fixme`'s in the code, it does make sense everyone's attention is brought to them -- so they can actually be fixed instead of silently ignored forever and ever until the end of times.

Reply:
At the moment, we can choose between:

1. Disabling the fixme check in pylint
2. Removing all `# FIXME`s in the code
3. Accepting that our Jenkins job fails all the time. 

Do you see a fourth option?

Reply:
4. take 3) and gradually fix what needs to be fixed.

Reply:
This would make the Jenkins basically useless. Do you mind, if we go with 1.) for the moment?